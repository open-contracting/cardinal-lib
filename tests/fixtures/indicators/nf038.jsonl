{"ocid":"0","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}]},{"status":"disqualified","tenderers":[{"id":"D"}]}]},"buyer":{"id":"B"},"tender":{"procuringEntity":{"id":"P"}}}
{"ocid":"1","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}]}]},"buyer":{"id":"1"},"tender":{"procuringEntity":{"id":"1"}}}
{"ocid":"2","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}]}]},"buyer":{"id":"2"},"tender":{"procuringEntity":{"id":"2"}}}
{"ocid":"3","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}]}]},"buyer":{"id":"3"},"tender":{"procuringEntity":{"id":"3"}}}
{"ocid":"4","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}]}]},"buyer":{"id":"4"},"tender":{"procuringEntity":{"id":"4"}}}
