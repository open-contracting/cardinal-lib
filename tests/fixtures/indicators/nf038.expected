{"OCID":{},"Buyer":{"B":{"NF038":0.5}},"ProcuringEntity":{"P":{"NF038":0.5}}}
